houses/views.py

```diff
@@ -1,3 +1,70 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db        import transaction
+
+from houses.models    import Country, Ghost, House, HouseImage, HouseType, City
+from core.s3_storages import s3_client
+from users.utils      import token_auth
+
+class BeingTheHostView(View):
+    @token_auth
+    @transaction.atomic
+    def post(self, request, *args, **kwargs):
+        try:
+            house_info   = request.POST.get("house_info")
+            house_images = request.FILES.getlist("house_images")
+
+            if len(house_images) < 2:
+                return JsonResponse({"message" : "At least two house_images are required"}, status=400)
+
+            data        = json.loads(house_info)
+            user        = request.user
+            house_type  = data["house_type"]
+            latitude    = data["latitude"]
+            longitude   = data["longitude"]
+            country     = data["country"]
+            city        = data["city"]
+            max_guest   = data["max_guest"]
+            trap        = data["trap"]
+            exit        = data["exit"]
+            ghost       = data["ghost"]
+            name        = data["name"]
+            description = data["description"]
+
+            house_type               = HouseType.objects.get(name=house_type)
+            ghost                    = Ghost.objects.get(name=ghost)
+            country, is_created_flag = Country.objects.get_or_create(name=country)
+            city, is_created_flag    = City.objects.get_or_create(
+                    name          = city, 
+                    country__name = country.name,
+                    country_id    = country.id
+                )
+            
+            house = House.objects.create(
+                name        = name,
+                description = description,
+                latitude    = latitude,
+                longitude   = longitude,
+                max_guest   = max_guest,
+                trap        = trap,
+                exit        = exit,
+                city        = city,
+                house_type  = house_type,
+                ghost       = ghost,
+                user        = user
+            )
+
+            for house_image in house_images:
+                folder_name    = 'house'
+                image_file_url = s3_client.upload(house_image, folder_name)
+                HouseImage(
+                    image_url = image_file_url,
+                    house     = House.objects.get(id=1)
```

Comment:
이미 line 45에서 House 정보를 db에 insert하고 house객체를 반환받아서 변수에 할당했습니다.
그대로 사용하면 되겠습니다.
```suggestion
                    house     = house.id
```

Reply:
테스트 과정에서 잠시 고쳤던 것을 체크하지 못하고 푸쉬했습니다.
다시 수정했습니다!